console-framework-client-spring-boot-starter/src/main/java/io/axoniq/console/framework/starter/AxoniqConsoleSpringProperties.java

```diff
@@ -28,6 +31,11 @@ public class AxoniqConsoleSpringProperties {
      * DLQ visibility.
      */
     private AxoniqConsoleDlqMode dlqMode = AxoniqConsoleDlqMode.FULL;
+    /**
+     * The list which can be used in combination with setting the {@code dlqMode} to
+     * {@link AxoniqConsoleDlqMode#LIMITED}. In that mode it will filter the diagnostics based on this list.
+     */
+    private List<String> dlqDiagnosticsWhitelist = new ArrayList<>();
```

Comment:
Just checking, but I assume the whitelist refers to the **keys** within the diagnostics, right?
If so, I think it would be good to clarify that point in the JavaDoc.